index.html

```diff
@@ -4,50 +4,80 @@
     <meta name="viewport" content="width=device-width">
     <title>Let's volleyball</title>
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic|Anonymous+Pro:400,700,400italic,700italic|Merriweather:400,700,300">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```

Comment:
Doesn't matter. Having source outside makes less impact on the server, having it locally is more reliable, however in case with boostrap CDN it shouldn't be an issue.